byzcoin/bcadmin/cmd_db.go

```diff
@@ -510,6 +514,23 @@ func dbCheck(c *cli.Context) error {
 				}
 			}
 		}
+
+		indexes := sb.GetFLIndexes()
+		for i, index := range indexes {
+			if index <= last.Index {
+				if i >= len(sb.ForwardLink) {
+					maxLink := len(indexes) - 1
+					for ; indexes[maxLink] > last.Index; maxLink-- {
+					}
```

Comment:
```suggestion

          // Get the index of the highest possible forwardlink based on the last
          // block.
					for ; indexes[maxLink] > last.Index; maxLink-- {
					}
```

Reply:
I removed that one and inserted it in the `for`-loop below.